base/linalg/matmul.jl

```diff
@@ -175,6 +175,14 @@ julia> Y
 """
 A_mul_B!(C::AbstractMatrix, A::AbstractVecOrMat, B::AbstractVecOrMat) = generic_matmatmul!(C, 'N', 'N', A, B)
 
+"""
+    A_mul_B!(A, B) -> C
```

Comment:
Perhaps remove ` -> C` since `C` is not needed in the docstring later?

Reply:
Removed. Thanks!